src/mumble/AudioInput.h

```diff
@@ -130,14 +140,23 @@ class AudioInputRegistrar {
 	const QString name;
 	int priority;
 
+	// A list of echo cancellation options available for this backend.
+	QList<EchoCancellationOption> echoOptions;
```

Comment:
`QVector` should be preferred in most cases (https://stackoverflow.com/questions/33609406/qlist-vs-qvector-revisited). Do you have a specific reason for choosing a `QList` instead?

And do you even make use of the Qt-specific functionality for this container? If not I would even suggest using `std::vector<...>`.
As we want to get rid of Qt in the backend eventually, things like that would help to prepare the way :point_up:

Reply:
I just see `QList` is used in many places in our code and thought it is a tradition. I can use `std` instead but was afraid that the code will be a messy combination of `Q` and `std`.

Reply:
> I can use std instead but was afraid that the code will be a messy combination of Q and std.

Yes this will probably end up being the case until we finally rework the old code and use std there as well. However the more small steps we do in changes like this, the less work it will end up being in the end :)

Reply:
I think I will do the entire replacing in a separate commit that gets rid of the Q-thing all at once.

Reply:
That would be even better :+1: